examples/player/simid_player.js

```diff
@@ -430,11 +453,12 @@ class SimidPlayer {
   
   /** The creative wants to play video. */
   onRequestPlay(incomingMessage) {
-    this.adVideoElement_.play().then(
-      // The play function returns a promise.
-      this.simidProtocol.resolve(incomingMessage),
-      this.simidProtocol.reject(incomingMessage)
-    );
+    
+    if (this.isLinearAd_) {
+      this.adVideoElement_.play().then(() => this.simidProtocol.resolve(incomingMessage));
+    } else {
+      this.simidProtocol.reject(incomingMessage);
+    }
```

Comment:
You still want to reject in the case where `this.adVideoElement_.play()` rejects (that happens more often then you would think -- for example, if the videoElement is currently trying to pause, if the source fails to load, etc.)